cineast-core/src/main/java/org/vitrivr/cineast/core/db/DBSelector.java

```diff
@@ -26,6 +27,8 @@
 
 public interface DBSelector extends Closeable {
 
+  Logger LOGGER = LogManager.getLogger();
```

Comment:
Logger is created but never used

Reply:
I beg to disagree: `CottontailSelector`, `PolyphenySelector` and `DataSource` all use this LOGGER.